src/server/game/World/IWorld.h

```diff
@@ -24,15 +24,17 @@
 #include "ObjectGuid.h"
 #include "QueryResult.h"
 #include "SharedDefines.h"
+#include "Unit.h"
```

Comment:
no need, just add `class Unit;`

Reply:
i will give it a try and test before pushing

Reply:
https://github.com/azerothcore/azerothcore-wotlk/pull/14199/commits/4347a2146082cb9d33181fba92809e5c7eb3123c

Reply:
had to revert due to build failure with world flow.
https://github.com/azerothcore/azerothcore-wotlk/pull/14199/commits/a0b7f3935085cde108bfda7e1de489a945d4e65c

Reply:
actually disregard that revert. appears there is a workflow issue with the custom modules.